tests/schema/tests.py

```diff
@@ -2832,8 +2835,7 @@ def test_func_index_unsupported(self):
     def test_func_index_nonexistent_field(self):
         index = Index(Lower('nonexistent'), name='func_nonexistent_idx')
         msg = (
-            "Cannot resolve keyword 'nonexistent' into field. Choices are: "
-            "height, id, name, uuid, weight"
```

Comment:
Why has this line been removed?

Reply:
I just added a field, but I'll fix the message, good point.